assets/js/components/PresenterStreamArea.tsx

```diff
@@ -12,12 +14,59 @@ type PresenterStreamAreaProps = {
 export const presenterStreams: { [key: string]: MediaStream } = {};
 let webrtc: MembraneWebRTC | null = null;
 
+export const changeIsTrackEnabled = (clientName: string, sourceType: SourceType, mode: boolean) => {
+  const track = presenterStreams[clientName].getTracks().find((track) => track.kind == sourceType);
+  if (track != undefined) {
+    track.enabled = mode;
+  }
+};
+
+const addMediaStreamTrack = (
+  clientName: string,
+  mediaStream: MediaStream,
+  sourceType: SourceType
+) => {
+  if (presenterStreams[clientName] == undefined) presenterStreams[clientName] = new MediaStream();
+  else {
+    const curTrack = presenterStreams[clientName]
+      .getTracks()
+      .find((track) => track.kind == sourceType);
+    if (curTrack != undefined) {
+      curTrack.stop();
+      presenterStreams[clientName].removeTrack(curTrack);
+    }
```

Comment:
Why do we remove tracks in `addMediaStreamTrack`?

Reply:
I don't want client MediaStream to have more than 1 track of each kind (audio/video), that wasn't tested tho (because of lack of more than 1 actual real devices).